utils/embedWithHooks.test.tsx

```diff
@@ -30,7 +30,7 @@ describe("EmbedWithHooks Component", () => {
     serviceAccountEmail: "test@example.com",
     serviceAccountPassword: "password",
     apiBaseUrl: "https://api.formant.io",
-    authTagSets: { role: "admin" },
+    authScope: { tags: { role: ["admin"] } },
```

Comment:
Is that `role: ["admin"]` a default value/example?
Wouldn't it be better to use a different example? role is confusing. Maybe `customer: ["acme"]`? Or something clearly more generic.

Reply:
yeah just a test.tsx